CovidWatch/libcontactrace/src/main/java/org/covidwatch/libcontactrace/cen/CENGenerator.kt

```diff
@@ -0,0 +1,8 @@
+package org.covidwatch.libcontactrace.cen
+
+/**
+ * CENGenerator Interface
+ */
+interface CENGenerator {
+    fun generate(): GeneratedCEN
```

Comment:
Is there any particular reason why we return a concrete implementation of the `CEN` interface here?